### ISSUE TITLE
Freesurfer=7.1.1-min

### ISSUE THREAD

**Issue by ConorOW** [ReproNim/neurodocker#457]
Hey folks,

I am interested in making a Freesurfer7-mini container.

Followed the instructions on the [examples page](https://www.repronim.org/neurodocker/user_guide/examples.html) on the website, but get the following error:
`[NEURODOCKER 2022-07-14 09:27:53,178 WARNING]: A newer version (0.8.0) of ReproNim/neurodocker is available. You are using 0.7.0
[NEURODOCKER 2022-07-14 09:27:53,184 ERROR]: Failed to instantiate <class 'neurodocker.interfaces.interfaces.FreeSurfer'>: URL not found for version '7.1.1-min'. Valid versions are 6.0.1, 6.0.0, 6.0.0-min`

Tried to then go back and get a more recent Neurodocker, hoping that would have the Freesurfer=7.1.1 but stumbled again.
`docker run --rm repronim/neurodocker:0.8.0 --help
Unable to find image 'repronim/neurodocker:0.8.0' locally
docker: Error response from daemon: manifest for repronim/neurodocker:0.8.0 not found: manifest unknown: manifest unknown.
See 'docker run --help'.`

Bit of a rookie, so grateful for any help.

Kindest regards,
Conor

**Comment by pwighton** [comment]
Hi @ConorOW, you can install the latest version of neurodocker, by cloning the repo, then running `pip install -e .` from the `neurodocker` directory.

You could also use `docker run pwighton/neurodocker:20220610`

**Comment by ConorOW** [comment]
Thanks so much Paul, works perfectly. Appreciate the support.

Cheers,
Conor

**Comment by kaczmarj** [comment]
@ConorOW - i'm working on getting the docker images in `repronim/neurodocker` up to date. for the moment, you can use

```
kaczmarj/neurodocker:0.9.1
```

thank you @pwighton for the help

related to #448

**Comment by ConorOW** [comment]
Thanks so much @kaczmarj, I will give that a run today.

Really appreciate your support.

Cheers,
Conor

**Comment by ConorOW** [comment]
Sorry @kaczmarj, have not been able to get things going...

Tried the following, as well as a number of little iterations with the base image...

`docker run --rm kaczmarj/neurodocker:0.9.1 --version`

neurodocker version 0.9.1+0.gad2f3a6.dirty

then:

`neurodocker generate docker --pkg-manager apt --base-image debian:buster-slim --freesurfer version=7.1.1-min > freesurfer7-min.Dockerfile`

then:

`docker build --tag freesurfer:7.1.1-min --file freesurfer7-min.Dockerfile .`

which gives me a big old error below.

Grateful for any advice, and I apologize for taking up more of your time. 

> [2/3] RUN apt-get update -qq     && apt-get install -y -q --no-install-recommends            bc            ca-certificates            curl            libgomp1            libxmu6            libxt6            perl            tcsh     && rm -rf /var/lib/apt/lists/*     && echo "Downloading FreeSurfer ..."     && mkdir -p /opt/freesurfer-7.1.1-min     && curl -fL https://dl.dropbox.com/s/c3earkfhhvdyuo4/freesurfer-7.1.1-min.tgz     | tar -xz -C /opt/freesurfer-7.1.1-min --owner root --group root --no-same-owner --strip-components 1          --exclude='average/mult-comp-cor'          --exclude='lib/cuda'          --exclude='lib/qt'          --exclude='subjects/V1_average'          --exclude='subjects/bert'          --exclude='subjects/cvs_avg35'          --exclude='subjects/cvs_avg35_inMNI152'          --exclude='subjects/fsaverage3'          --exclude='subjects/fsaverage4'          --exclude='subjects/fsaverage5'          --exclude='subjects/fsaverage6'          --exclude='subjects/fsaverage_sym'          --exclude='trctrain':
#5 0.720 W: Failed to fetch http://deb.debian.org/debian/dists/buster/InRelease  Connection failed [IP: 146.75.94.132 80]
#5 0.720 W: Failed to fetch http://deb.debian.org/debian-security/dists/buster/updates/InRelease  Connection failed [IP: 146.75.94.132 80]
#5 0.720 W: Failed to fetch http://deb.debian.org/debian/dists/buster-updates/InRelease  Connection failed [IP: 146.75.94.132 80]
#5 0.721 W: Failed to fetch http://neuro.debian.net/debian/dists/buster/InRelease  Connection failed [IP: 129.170.233.11 80]
#5 0.721 W: Failed to fetch http://neuro.debian.net/debian/dists/data/InRelease  Connection failed [IP: 129.170.233.11 80]
#5 0.721 W: Some index files failed to download. They have been ignored, or old ones used instead.
#5 0.731 Reading package lists...
#5 0.738 Building dependency tree...
#5 0.738 Reading state information...
#5 0.739 Package perl is not available, but is referred to by another package.
#5 0.739 This may mean that the package is missing, has been obsoleted, or
#5 0.739 is only available from another source
#5 0.739 However the following packages replace it:
#5 0.739   perl-base
#5 0.739 
#5 0.739 Package ca-certificates is not available, but is referred to by another package.
#5 0.739 This may mean that the package is missing, has been obsoleted, or
#5 0.739 is only available from another source
#5 0.739 
#5 0.740 E: Unable to locate package bc
#5 0.740 E: Package 'ca-certificates' has no installation candidate
#5 0.740 E: Unable to locate package curl
#5 0.740 E: Unable to locate package libgomp1
#5 0.740 E: Unable to locate package libxmu6
#5 0.740 E: Unable to locate package libxt6
#5 0.740 E: Package 'perl' has no installation candidate
#5 0.740 E: Unable to locate package tcsh
------
executor failed running [/bin/sh -c apt-get update -qq     && apt-get install -y -q --no-install-recommends            bc            ca-certificates            curl            libgomp1            libxmu6            libxt6            perl            tcsh     && rm -rf /var/lib/apt/lists/*     && echo "Downloading FreeSurfer ..."     && mkdir -p /opt/freesurfer-7.1.1-min     && curl -fL https://dl.dropbox.com/s/c3earkfhhvdyuo4/freesurfer-7.1.1-min.tgz     | tar -xz -C /opt/freesurfer-7.1.1-min --owner root --group root --no-same-owner --strip-components 1          --exclude='average/mult-comp-cor'          --exclude='lib/cuda'          --exclude='lib/qt'          --exclude='subjects/V1_average'          --exclude='subjects/bert'          --exclude='subjects/cvs_avg35'          --exclude='subjects/cvs_avg35_inMNI152'          --exclude='subjects/fsaverage3'          --exclude='subjects/fsaverage4'          --exclude='subjects/fsaverage5'          --exclude='subjects/fsaverage6'          --exclude='subjects/fsaverage_sym'          --exclude='trctrain']: exit code: 100

**Comment by pwighton** [comment]
Hi @ConorOW,

Can you try replacing the command:
```
neurodocker generate docker --pkg-manager apt --base-image debian:buster-slim --freesurfer version=7.1.1-min > freesurfer7-min.Dockerfile
```

with:
```
docker run --rm kaczmarj/neurodocker:0.9.1 generate docker --pkg-manager apt --base-image debian:buster-slim --freesurfer version=7.1.1-min > freesurfer7-min.Dockerfile
```

**Comment by ConorOW** [comment]
Thanks so much @pwighton, I got it to run using your suggestion. I would then get an error at the build stage. 

So this wouldn't work:
`docker build --tag freesurfer:7.1.1-min --file freesurfer7-min.Dockerfile .`

but this would:
`docker build --file freesurfer7-min.Dockerfile .`

So I took the second approach, and then named/tagged the image:
`docker image tag <image id> freesurfer7-min`

Thank you again for your wonderful help.

Cheers,
Conor
